chirp/drivers/uvk5.py

```diff
@@ -0,0 +1,1339 @@
+                RadioSettingValueList(
+                    REMENDOFTALK_LIST,
+                    REMENDOFTALK_LIST[tmpalarmmode]))
+        basic.append(rs)
+
+        # Repeater tail tone elimination
+        tmprte = _mem.repeater_tail_elimination
+        if tmprte >= len(RTE_LIST):
+            tmprte = 0
+        rs = RadioSetting(
+                "repeater_tail_elimination",
+                "Repeater tail tone elimination",
+                RadioSettingValueList(RTE_LIST, RTE_LIST[tmprte]))
+        basic.append(rs)
+
+        # Logo string 1
+        logo1 = str(_mem.logo_line1).strip("\x20\x00\xff")  # +"\x20"*12
+        logo1 = logo1[0:12]
+        rs = RadioSetting("logo1", "Logo string 1 (12 characters)",
+                          RadioSettingValueString(0, 12, logo1))
+        basic.append(rs)
+
+        # Logo string 2
+        logo2 = str(_mem.logo_line2).strip("\x20\x00\xff")  # +"\x20"*12
+        logo2 = logo2[0:12]
+        rs = RadioSetting("logo2", "Logo string 2 (12 characters)",
+                          RadioSettingValueString(0, 12, logo2))
+        basic.append(rs)
+
+        # unlock settings
+
+        # F-LOCK
+        tmpflock = _mem.int_flock
+        if tmpflock >= len(FLOCK_LIST):
+            tmpflock = 0
+        rs = RadioSetting(
+            "flock", "F-LOCK",
+            RadioSettingValueList(FLOCK_LIST, FLOCK_LIST[tmpflock]))
+        unlock.append(rs)
+
+        # 350TX
+        rs = RadioSetting("350tx", "350TX", RadioSettingValueBoolean(
+            bool(_mem.int_350tx > 0)))
+        unlock.append(rs)
+
+        # unknown1
+        rs = RadioSetting("unknown11", "UNKNOWN1",
+                          RadioSettingValueBoolean(
+                              bool(_mem.int_unknown1 > 0)))
+        unlock.append(rs)
+
+        # 200TX
+        rs = RadioSetting("200tx", "200TX", RadioSettingValueBoolean(
+            bool(_mem.int_200tx > 0)))
+        unlock.append(rs)
+
+        # 500TX
+        rs = RadioSetting("500tx", "500TX", RadioSettingValueBoolean(
+            bool(_mem.int_500tx > 0)))
+        unlock.append(rs)
+
+        # 350EN
+        rs = RadioSetting("350en", "350EN", RadioSettingValueBoolean(
+            bool(_mem.int_350en > 0)))
+        unlock.append(rs)
+
+        # SCREEN
+        rs = RadioSetting("screen", "SCREEN", RadioSettingValueBoolean(
+            bool(_mem.int_screen > 0)))
+        unlock.append(rs)
+
+        # readonly info
+        # Firmware
+        if self.FIRMWARE_VERSION == "":
+            firmware = "To get the firmware version please download"
+            "the image from the radio first"
+        else:
+            firmware = self.FIRMWARE_VERSION
+
+        val = RadioSettingValueString(0, 128, firmware)
+        val.set_mutable(False)
+        rs = RadioSetting("fw_ver", "Firmware Version", val)
+        roinfo.append(rs)
+
+        # Driver version
+        val = RadioSettingValueString(0, 128, DRIVER_VERSION)
```

Comment:
Please remove per above.

Reply:
As commented above, will be removed once this driver gets into chirp